source/_posts/2024-08-21-hacs-the-best-way-to-share-community-made-projects.markdown

```diff
@@ -0,0 +1,113 @@
+
+**Scripts and Templates** - This is the easiest way to share scripts and templates, while also adding the ability to push new improvements to users via updates.
+
+## A Brief History of HACS
+
+HACS started in 2019 as my ([Joakim Sørensen](https://github.com/ludeeus) ) third attempt to build a community store. It started small and quickly became the de facto standard, with so many talented community creators submitting their projects. The Home Assistant team was supportive from day one, and even as HACS gained hundreds of thousands of downloads, we all agreed it should remain an optional addition to Home Assistant. Giving users the choice to add new features to Home Assistant at the cost of stability.
+
+I joined [Nabu Casa](https://www.nabucasa.com/) in 2020, partly due to the work I did on HACS, but I was hired to work on Home Assistant more generally, not to work on HACS. Incidentally, I'm not the only developer working at Nabu Casa who got their start on HACS: [Paul Bottein](https://github.com/piitaya)  on the frontend team built [Mushroom Cards](https://github.com/piitaya/lovelace-mushroom), and  [Marcel van der Veldt](https://github.com/marcelveldt) who currently supports Matter development built [Music Assistant](https://music-assistant.io/).
+
+Over the years, I continued to work on HACS in my spare time, but there were always aspects of the original version that limited its ability to grow. Recently, the [Open Home Foundation](https://www.openhomefoundation.org/) reached out to make HACS a collaboration partner, and provided development support. Numerous people were brought in to support, from across Nabu Casa and the community, leading to the large list of improvements you see today.
+
+Just before we released this big update, it passed 5,000 stars on GitHub! It's the projects on HACS that make it what it is today, and I'd like to sincerely thank everyone who continues to contribute their code onto the store.
+
+## How to install
+
+{% note %}
+**Proceed with caution** - HACS is not for everyone.  If you're a Home Assistant beginner or are looking for the most stable experience in your smart home, HACS is probably not for you. These are community-made projects that do not receive the same rigorous reviews required of projects submitted to Home Assistant. There may be projects that don't work or, even worse, break your system. Unlike add-ons that run separately and alongside Home Assistant, HACS projects run their code directly in Home Assistant, which increases the chances of crashing or harming your system. Before using HACS, always make a [backup of your system](/integrations/backup/) and download that backup to another device.
```

Comment:
```suggestion
**Proceed with caution** - HACS is not for everyone.  If you're a Home Assistant beginner or are looking for the most stable experience in your smart home, HACS is probably not for you. These are community-made projects that do not receive the same rigorous reviews required of projects submitted to Home Assistant. There may be projects that don't work or, even worse, break your system. Unlike add-ons that run separately and alongside Home Assistant, HACS projects run their code directly in Home Assistant, which increases the chances of crashing or harming your system. Before using HACS, always make a [backup of your system](https://hacs.xyz/docs/use/data/#creating-a-backup) and download that backup to another device.
```
The backup integration is for Core and Container. Should we link to the HACS backup docs instead?